src/System.Runtime.WindowsRuntime.UI.Xaml/pkg/build/net45/System.Runtime.WindowsRuntime.UI.Xaml.targets

```diff
@@ -0,0 +1,18 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Choose>
+    <When Condition="'$(DoNotReferenceWinRT)' != 'true'">
+      <PropertyGroup>
+        <!-- include the reference to System.Runtime -->
+        <_HasReferenceToSystemRuntime>true</_HasReferenceToSystemRuntime>
+      </PropertyGroup>
+      <ItemGroup>
+        <ReferencePath Include="$(MSBuildThisFileDirectory)..\..\ref\netstandard1.0\System.Runtime.WindowsRuntime.UI.Xaml.dll">
+          <!-- Private = false to make these reference only -->
+          <Private>false</Private>
+          <!-- hide these from Assemblies view in Solution Explorer, they will be shown under packages -->
```

Comment:
I believe they won't be shown at all since you aren't adding the NuGetPackage* metadata.  I think that's OK but you may want to update the comment.